Formula/bic.rb

```diff
@@ -0,0 +1,40 @@
+class Bic < Formula
+  desc "C interpreter and API explorer"
+  homepage "https://github.com/hexagonal-sun/bic"
+  url "https://github.com/hexagonal-sun/bic/releases/download/v1.0.0/bic-v1.0.0.tar.gz"
+  sha256 "553324e39d87df59930d093a264c14176d5e3aaa24cd8bff276531fb94775100"
+  head do
+    url "https://github.com/hexagonal-sun/bic.git"
+
+    depends_on "autoconf-archive" => :build
+    depends_on "automake" => :build
+    depends_on "bison" => :build # macOS bison is too outdated, build fails unless gnu bison is used
+    depends_on "libtool" => :build if build.head?
+
+    uses_from_macos "flex" => :build
+  end
+
+  depends_on "autoconf" => :build
```

Comment:
I don't think `autoconf` should be required when building from the release tarball.

Reply:
OK, I'll give that a shot and test locally. Thanks for taking a look!

Reply:
Local build worked fine, so as long as CI agrees, then I think we're ready to merge now.